litex/tools/litex_json2dts.py

```diff
@@ -133,6 +133,7 @@ def generate_dts(d):
             bus-frequency  = <{sys_clk_freq}>;
             compatible = "simple-bus";
             ranges;
+            interrupt-parent = <&intc0>;
```

Comment:
The interrupt controller is global for the soc in both of our supported systems, so it should go at the top level. This way it is inherited by all nodes below it, ie, all of them.

Reply:
Alright, I can move it.

Reply:
> Alright, I can move it.

Done. And tested.